### DISCUSSION TITLE
How to serialize Prisma Object in NestJS?

### DISCUSSION THREAD

**Discussion by ronyfhebrian** [prisma/prisma#9233] [Q&A]
I have tried using Class Transformer, but it doesn't make any sense since Prisma doesn't need Entity, and Prisma Type cannot be Exclude() Is there any way to exclude key from Prisma object e.g. createdAt or password? Thank you

**Comment by ryands17** [comment]
No idea about NestJS, but you can use [`select`](https://www.prisma.io/docs/concepts/components/prisma-client/select-fields#select-specific-fields) in Prisma to only pass the required fields to the client.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
